perk-store-services/src/main/java/org/exoplatform/perkstore/service/PerkStoreService.java

```diff
@@ -316,6 +322,16 @@ public Product saveProduct(Product product, String username) throws Exception {
     return orders;
   }
 
+  public Long countOrders(String username, OrderFilter filter) {
```

Comment:
```suggestion
  public Long countOrders(OrderFilter filter, String username) {
```
Same params order of getOrders method